src/metabase/models/serialization.clj

```diff
@@ -391,6 +391,17 @@
 (defmethod descendants :default [_ _]
   nil)
 
+(defmulti ascendants
+  "Return set of `[model-name database-id]` pairs for all entities containing this entity, required to successfully
+  load this entity in destination db.
+
+  Dispatched on model-name."
```

Comment:
The wording of this docstring is insufficiently precise. I think it needs to explain that the set needs to include a full set of parent nodes up to the root, or something like that. Because a grandparent entity doesn't necessarily "contain" the entity of a grandchildren. At least not for all uses of the word "contain".